packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -153,6 +154,15 @@ const App = (props) => {
         location?.pathname
     )
 
+    // TODO: Remove test logs before merging
+    if (isServer) {
+        logger.log('log test message')
+        logger.info('info test message', {key:'retail-react-app', details:['App']})
+        logger.warn('warn test message', {key:'retail-react-app', details:['App']})
+        logger.error('error test message', {key:'retail-react-app', details:['App']})
+        logger.debug('debug test message', {key:'retail-react-app', details:['App']})
+    }
```

Comment:
What would the effect of **not** having this condition be? What is the "logger" when you are on the client, and why did you choose not to use it?

Reply:
The logger only adds namespacing to logs using the regular `console` object. If we remove the condition, we will also see the logs in the browser console. The condition is there to focus on filtering server-side logs without polluting the browser console.